app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+	# Returns the full title on a per-page basis.
+  def full_title(page_title = '')
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```

Comment:
có thể sử dụng toán tử điều kiện, vì nó chỉ có 2 trường hợp: empty và !empty.
điều kiện ? kết quả 1 : kết quả 2